platform/node/test/suite_implementation.js

```diff
@@ -65,6 +68,26 @@ module.exports = function (style, options, callback) {
                 applyOperations(operations.slice(1), callback);
             });
 
+        } else if (operation[0] === 'addImage') {
+            console.log(path.join(__dirname, '../../../node_modules', 'mapbox-gl-test-suite', operation[2]));
+            var img = PNG.sync.read(fs.readFileSync(path.join(__dirname, '../../../node_modules', 'mapbox-gl-test-suite', operation[2])));
+
+            map.addImage(operation[1], img.data, {
+                height: img.height,
+                width: img.width,
+                pixelRatio: 1
+            });
+
+            map.addLayer({
```

Comment:
This should be test-specific; that is the [add-image test](https://github.com/mapbox/mapbox-gl-test-suite/compare/add-image) should use:

```
      "operations": [
        [
          "addImage",
          "marker",
          "./image/marker.png"
        ],
        [
          "addLayer",
          ...
        ],
        [
          "wait"
        ]
      ]
```